WalletWasabi.Fluent/Views/AddWalletPageView.axaml

```diff
@@ -6,65 +6,76 @@
              xmlns:c="clr-namespace:WalletWasabi.Fluent.Controls"
              xmlns:conv="clr-namespace:WalletWasabi.Fluent.Converters"
              xmlns:i="clr-namespace:Avalonia.Xaml.Interactivity;assembly=Avalonia.Xaml.Interactivity"
+             xmlns:converters="clr-namespace:WalletWasabi.Fluent.Converters;assembly=WalletWasabi.Fluent"
              xmlns:behaviors="clr-namespace:WalletWasabi.Fluent.Behaviors"
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
              x:DataType="vm:HomePageViewModel"
              x:Class="WalletWasabi.Fluent.Views.AddWalletPageView">
-    <c:ContentArea Title="Add Wallet" Caption="Type in your wallet's name and select an option below:">
-        <DockPanel>
-            <StackPanel DockPanel.Dock="Top" Margin="0 0 0 40">
-                <Label Content="Wallet _Name:" Target="walletNameTb" />
-                <TextBox x:Name="walletNameTb" Text="{Binding WalletName}" Watermark="(e.g. My Wallet)">
-                    <i:Interaction.Behaviors>
-                        <behaviors:FocusOnAttachedBehavior />
-                    </i:Interaction.Behaviors>
-                </TextBox>
+  <UserControl.Resources>
+    <converters:BoolToOpacityConverter x:Key="BoolToOpacityConverter" />
+    <converters:OpacityToBoolConverter x:Key="OpacityToBoolConverter" />
+  </UserControl.Resources>
+
+  <c:ContentArea Title="Add Wallet" Caption="Type in your wallets name and select an option:">
+    <DockPanel>
+      <StackPanel DockPanel.Dock="Top" Margin="0 0 0 40">
+        <Label Content="Wallet _Name:" Target="walletNameTb" />
+        <TextBox x:Name="walletNameTb" Text="{Binding WalletName}" Watermark="(e.g. My Wallet)">
+          <i:Interaction.Behaviors>
+            <behaviors:FocusOnAttachedBehavior />
+          </i:Interaction.Behaviors>
+        </TextBox>
+      </StackPanel>
+      <ScrollViewer>
+        <StackPanel Spacing="10" Margin="0 0 0 40">
+          <StackPanel.Styles>
+            <Style Selector="Button">
+              <Setter Property="HorizontalAlignment" Value="Stretch" />
+              <Setter Property="BorderThickness" Value="1" />
+              <Setter Property="Padding" Value="0" />
+              <Setter Property="IsEnabled" Value="{Binding OptionsEnabled}" />
+              <Setter Property="Opacity" Value="{Binding OptionsEnabled, Converter={StaticResource BoolToOpacityConverter}}" />
+              <Setter Property="IsVisible" Value="{Binding $self.Opacity, Converter={StaticResource OpacityToBoolConverter}}}" />
```

Comment:
nice :)

Reply:
yes clever...! but do we actually need IsVisible to be false.. opacity = 0 cant be seen :D

Reply:
i guess so it's not hit-testable?

Reply:
Thanks 🚀